src/js/components/explorer/detail/header/RootHeader.jsx

```diff
@@ -19,35 +21,79 @@ const propTypes = {
     lastUpdate: PropTypes.string
 };
 
-const RootHeader = (props) => {
-    const type = sidebarTypes[props.root];
-    return (
-        <div className="detail-header">
+export default class RootHeader extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showInfoTooltip: false
+        };
+
+        this.showTooltip = this.showTooltip.bind(this);
+        this.closeTooltip = this.closeTooltip.bind(this);
+    }
+
+    showTooltip() {
+        this.setState({
+            showInfoTooltip: true
+        });
+    }
+
+    closeTooltip() {
+        this.setState({
+            showInfoTooltip: false
+        });
+    }
+
+    render() {
+        let tooltip = null;
+        if (this.state.showInfoTooltip) {
+            tooltip = (
+                <ExplorerInfoToolTip
+                    closeTooltip={this.closeTooltip} />
+            );
+        }
+
+        const type = sidebarTypes[this.props.root];
+        const header = (
             <div className="detail-header__labels">
                 <h2 className="detail-header__title">
-                    You are viewing FY {props.fy} spending
-                    by <span className="detail-header__title detail-header__title_capitalize">{type}</span>
+            You are viewing FY {this.props.fy} spending
```

Comment:
Could you fix the indentation in this section? The content of the `h2` and `div` should be one tab in from their opening and closing element tags.